activerecord/test/cases/enum_test.rb

```diff
@@ -378,6 +378,14 @@ class EnumTest < ActiveRecord::TestCase
     assert_equal "published", @book.status
   end
 
+  test "attributes_for_database" do
+    assert_equal 2, @book.attributes_for_database["status"]
+
+    @book.status = "published"
+
+    assert_equal 2, @book.attributes_for_database["status"]
+  end
```

Comment:
This might be overkill, but I added it here since the test above for `attribute_for_database` is very similar and I wanted to follow the existing pattern.